test/tests/test_fn_update/test_resource_change.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source $(dirname $0)/fnupdate_utils.sh
+
+ROOT=$(dirname $0)/../../..
+
+env=python-$(date +%N)
+fn=hellopython-$(date +%N)
+
+mincpu1=40
+maxcpu1=140
+minmem1=256
+maxmem1=512
+
+mincpu2=80
+maxcpu2=200
+minmem2=512
+maxmem2=768
+
+log "Creating Python env $env"
+fission env create --name $env --image fission/python-env --mincpu 20 --maxcpu 100 --minmemory 128 --maxmemory 256
+trap "fission env delete --name $env" EXIT
+
+log "Creating function $fn"
+fission fn create --name $fn --env $env --code $ROOT/examples/python/hello.py --minscale 1 --maxscale 4 --executortype newdeploy --mincpu $mincpu1 --maxcpu $maxcpu1 --minmemory $minmem1 --maxmemory $maxmem1
+
+log "Creating route"
+fission route create --function ${fn} --url /${fn} --method GET
+
+log "Waiting for updates to take effect"
+sleep 5
+
+#If variable not used, shell assumes 'function' to be a real function
+func=function
+
+maxcpu_actual=$(kubectl describe $func $fn -n default|grep Cpu|head -1|cut -f 2|tr -dc '0-9')
```

Comment:
would it be better to use `kubectl` with `-o jsonpath` to extract the value of a specific field in the object? ex: to find controller_port from the controller pod spec, something like this could be done controller_port=`kubectl get svc controller -n $FISSION_NAMESPACE -ojsonpath='{.spec.ports[0].targetPort}'`.
that way we can avoid the cut with picking specific fields.